purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -22,12 +25,15 @@ data class PaywallData(
     /**
      * The base remote URL where assets for this paywall are stored.
      */
-    val assetBaseURL: URL,
-    internal val localization: Map<String, LocalizedConfiguration>,
+    @SerialName("asset_base_url") @Serializable(with = URLSerializer::class) val assetBaseURL: URL,
+    // TODO-PAYWALLS: do we need this one?
+    @SerialName("default_locale") val defaultLocale: String,
```

Comment:
iOS doesn't have this property, but it's returned in the JSON

cc @NachoSoto

Reply:
Hmm we should confirm with the team, but maybe we should make it optional in case it disappears?

Reply:
Yeah this is no longer required and will be removed from the backend. It's only still there for backwards compatibility with Josh's app 😅

Reply:
Lol ok. I'll remove it then